### DISCUSSION TITLE
Does Harbor is exposed to https://curl.se/docs/CVE-2023-38545.html ?

### DISCUSSION THREAD

**Discussion by djebita** [goharbor/harbor#19726] [Q&A]
Dears Harbor developers community,

Could you please check based on the description of this curl CVE https://curl.se/docs/CVE-2023-38545.html
if Harbor is exposed or not regarding this one ?

If Harbor is using curl/libcurl, could you please share which release version of Harbor got the patch 8.4.0 for it ?

Thank you in advance.

**Comment by stonezdj** [comment]
Replied in security mail group
